addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java

```diff
@@ -285,7 +286,28 @@ protected void updateThingFromDevice(Thing thing, AVMFritzBaseModel device) {
                 updateThingChannelState(thing, CHANNEL_BATTERY_LOW, UnDefType.UNDEF);
             } else {
                 updateThingChannelState(thing, CHANNEL_BATTERY_LOW,
-                        HeatingModel.BATTERY_ON.equals(device.getHkr().getBatterylow()) ? OnOffType.ON : OnOffType.OFF);
+                        BATTERY_ON.equals(device.getHkr().getBatterylow()) ? OnOffType.ON : OnOffType.OFF);
+            }
+        }
+        if (device instanceof DeviceModel && device.isAlarmSensor() && ((DeviceModel) device).getAlert() != null) {
+            updateThingChannelState(thing, CHANNEL_CONTACT_STATE,
+                    AlertModel.ON.equals(((DeviceModel) device).getAlert().getState()) ? OpenClosedType.OPEN
```

Comment:
Optional Why not adding the the static imports also here

Reply:
I left `AlertModel.ON` non static to prevent misunderstandings. You remember `OnOffType.ON`?